client/my-sites/customer-home/locations/primary/index.jsx

```diff
@@ -6,26 +6,24 @@ import React, { useEffect } from 'react';
 /**
  * Internal dependencies
  */
-import ConnectAccounts from 'my-sites/customer-home/cards/tasks/connect-accounts';
-import Webinars from 'my-sites/customer-home/cards/tasks/webinars';
-import FindDomain from 'my-sites/customer-home/cards/tasks/find-domain';
-import SiteSetupList from 'my-sites/customer-home/cards/tasks/site-setup-list';
-import SiteSetupListEcommerce from 'my-sites/customer-home/cards/tasks/site-setup-checklist-ecommerce';
-import DeprecateEditor from 'my-sites/customer-home/cards/tasks/deprecate-editor';
```

Comment:
The change got masked a little here as I had to fix the new eslint rule about relative imports. The main thing is that this `DeprecateEditor` import has been removed.